itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/universal/methods/AbstractQuarkusTestMethods.java

```diff
@@ -37,11 +43,11 @@
  * @author olkornii@redhat.com
  *
  */
-public abstract class AbstractCreateNewProjectTest {
+public abstract class AbstractQuarkusTestMethods {
```

Comment:
I would change it to AbstractQuarkusTest, because it is a common class for all quarkus tests....

Reply:
rdy